test/runtests.jl

```diff
@@ -459,6 +459,17 @@ end
     rootfile = ROOTFile(joinpath(SAMPLES_DIR, "issue61.root"))
     @test rootfile["Events/Jet_pt"][:] == Vector{Float32}[[], [27.324587, 24.889547, 20.853024], [], [20.33066], [], []]
     close(rootfile)
+
+    # issue 78
+    rootfile = ROOTFile(joinpath(SAMPLES_DIR, "issue61.root"))
+    arr = LazyTree(rootfile,"Events").Jet_pt;
+    _ = length.(arr);
+    if length(arr.buffer) == Threads.nthreads()
```

Comment:
surely this branch is already determined right?

Reply:
Ah, let me simplify it. I did it at the time because another PR changed the type of the buffer to be a vector with length nthreads().